substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -530,6 +541,31 @@ pub mod pallet {
 
 			Ok(Pays::No.into())
 		}
+
+		/// Set the parameters.
+		///
+		/// - `origin`: An origin complying with `ParamsOrigin` or root.
+		/// - `params`: The new parameters for the pallet.
+		#[pallet::weight(T::WeightInfo::set_params())]
+		#[pallet::call_index(9)]
+		pub fn set_partial_params(
+			origin: OriginFor<T>,
+			partial_params: Box<PartialParamsOf<T, I>>,
+		) -> DispatchResult {
+			T::ParamsOrigin::ensure_origin_or_root(origin)?;
+			Params::<T, I>::mutate(|p| {
+				override_params!(p.active_salary, partial_params.active_salary);
+				override_params!(p.passive_salary, partial_params.passive_salary);
+				override_params!(p.demotion_period, partial_params.demotion_period);
+				override_params!(p.min_promotion_period, partial_params.min_promotion_period);
+				if partial_params.offboard_timeout.is_some() {
+					p.offboard_timeout = partial_params.offboard_timeout.unwrap();
+				}
```

Comment:
```suggestion
				if let Some(offboard_timeout) = partial_params.offboard_timeout {
					p.offboard_timeout = offboard_timeout;
				}
```

if you see a unwrap in the code, it probably means the code can be improved

Reply:
Resolved in https://github.com/paritytech/polkadot-sdk/pull/3843/commits/64ca32d16f80afee195a57a10f9142ea1ccb1ad5